bootstrap-tema.html

```diff
@@ -0,0 +1,127 @@
+        <div class="container-fluid">
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+            <img
+                src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcROhl2i_y2PCQltDB79sm_z1LDBjDOn_yL1Og&usqp=CAU" />
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+            <a href="file:///E:/CodeCool/html/bootstraptable.html" class="butoane btn-light btn-link">Echipa</a>
+        </div>
+    </nav>
+
+
+    <div class="container-fluid" style="height: 250px;">
+        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
+            <ol class="carousel-indicators">
+                <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+                <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+                <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+            </ol>
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                    <img src="https://wallpapercave.com/wp/wp1810626.png" class="poze d-block w-100" alt="...">
+                </div>
+                <div class="carousel-item">
+                    <img src="https://images8.alphacoders.com/746/thumb-1920-746199.jpg" class="poze d-block w-100"
+                        alt="...">
+                </div>
+                <div class="carousel-item">
+                    <img src="https://images.hdqwalls.com/wallpapers/fsociety-mr-robot-4n.jpg"
+                        class="poze d-block w-100 " alt="...">
+                </div>
+            </div>
+            <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
+                <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                <span class="sr-only">Previous</span>
+            </a>
+            <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+                <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                <span class="sr-only">Next</span>
+            </a>
+        </div>
+
+        <nav class="navbar navbar-default" style="padding: 0px;">
+            <div class="container-fluid yoyo">
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSgNdaFoojoZVBkB4hDsuunqfVggqwbEvB3Og&usqp=CAU"
+                    class="case col-4" />
```

Comment:
To make these images more visible on small displays, you could use a col-12 , col-md-4